projects/common/src/utilities/validators/regex-validator.ts

```diff
@@ -0,0 +1,10 @@
+export const isValidRegex = (regex: string) => {
```

Comment:
add appropriate comments at the top please.

Reply:
updated